apps/new/widget/page/projects/Editor.jsx

```diff
@@ -271,6 +271,40 @@ const handleTags = (tags) => {
   setTags(filtered);
 };
 
+// Handle NEAR Catalog Projects
+const catalogProjectId = props.catalogProjectId;
+const catalogProjectData = null;
+if (catalogProjectId) {
+  const indexer = "https://nearcatalog.xyz/wp-json/nearcatalog/v1";
+  const query = "";
+  query = fetch(indexer + "/project?pid=" + catalogProjectId);
+  if (!query.body) {
+    return <>No data Found!</>;
+  }
```

Comment:
we can add this to project-data file, like we have one for fetching project from social db, for potlock I think it's their SDK fn that's used.

Reply:
Where?

Reply:
https://github.com/NEARBuilders/gateway/blob/develop/apps/new/widget/lib/projects.jsx#L69
Like this
we can have one for project from near catalog